packages/lesswrong/components/review/FrontpageReviewPhase.tsx

```diff
@@ -139,7 +138,7 @@ const FrontpageReviewPhase = ({classes}) => {
         </LWTooltip>}
       >
         <LWTooltip title="All Posts written in 2019 are eligible to participate in the review. Click here to see all posts written in 2019.">
-          <Link to={"/allPosts?timeframe=yearly&after=2019-01-01&before=2020-01-01&limit=100&sortedBy=top"}><SettingsButton showIcon={false} label="See All 2019 Posts"/></Link>
+          <Link to={"/allPosts?timeframe=yearly&after=2019-01-01&before=2020-01-01&limit=100&sortedBy=top?filter=unnominated2019"}><SettingsButton showIcon={false} label="Nominate 2019 Posts"/></Link>
```

Comment:
This should be `&filter=` not `?filter=` (the rule is that the first URL parameter is a `?` and all subsequent ones are `&`).

Reply:
I think you probably also want to add this as a filter option in `PostsListSettings`. I'm not sure what happens if you don't.

Reply:
"untagged" and other weird one-of filters don't appear there. I think that's just for making it show up super legibly in the allPosts page, which I don't think we want.